lib/core/presentation/pages/event/guest_event_detail_page/widgets/host_event_detail_config_grid.dart

```diff
@@ -0,0 +1,246 @@
+        crossAxisCount: 3,
+        crossAxisSpacing: 10,
+        mainAxisSpacing: 10,
+        childAspectRatio: 0.95,
+      ),
+      delegate: SliverChildBuilderDelegate(
+        (BuildContext context, int index) {
+          return GridItemWidget(
+            item: listData[index],
+          );
+        },
+        childCount: 6, // Total number of items
+      ),
+    );
+  }
+}
+
+class GridItemWidget extends StatelessWidget {
+  final EventConfigGridViewModel? item;
+
+  const GridItemWidget({
+    super.key,
+    required this.item,
+  });
+
+  @override
+  Widget build(BuildContext context) {
+    final colorScheme = Theme.of(context).colorScheme;
+    return Container(
+      padding: const EdgeInsets.only(top: 1, left: 1, right: 1, bottom: 6),
+      decoration: ShapeDecoration(
+        color: colorScheme.secondaryContainer,
+        shape: RoundedRectangleBorder(
+          borderRadius: BorderRadius.circular(16),
+        ),
+      ),
+      child: Container(
+        padding: const EdgeInsets.all(15),
+        clipBehavior: Clip.antiAlias,
+        decoration: ShapeDecoration(
+          gradient: LinearGradient(
+            begin: const Alignment(0.00, -1.00),
+            end: const Alignment(0, 1),
+            colors: [colorScheme.secondaryContainer, Colors.black],
+          ),
+          shape: RoundedRectangleBorder(
+            borderRadius: BorderRadius.circular(15),
+          ),
+        ),
+        child: Column(
+          mainAxisAlignment: MainAxisAlignment.spaceBetween,
+          crossAxisAlignment: CrossAxisAlignment.start,
+          children: [
+            item?.icon ?? const SizedBox(),
+            SizedBox(
+              width: double.infinity,
+              child: Column(
+                mainAxisSize: MainAxisSize.min,
+                mainAxisAlignment: MainAxisAlignment.start,
+                crossAxisAlignment: CrossAxisAlignment.start,
+                children: [
+                  SizedBox(
+                    width: double.infinity,
+                    child: Text(
+                      item?.title ?? '',
+                      style: Typo.small.copyWith(
+                        fontFamily: FontFamily.nohemiVariable,
+                        color: Colors.white,
+                        fontWeight: FontWeight.w700,
+                      ),
+                    ),
+                  ),
+                  // const SizedBox(height: 6),
+                  item!.showProgressBar == true
+                      ? Padding(
+                          padding: EdgeInsets.symmetric(vertical: 6.h),
+                          child: Container(
+                            width: 79,
+                            height: 2,
+                            decoration: ShapeDecoration(
+                              color: colorScheme.outline,
+                              shape: RoundedRectangleBorder(
+                                borderRadius: BorderRadius.circular(100),
```

Comment:
why radius 100 ?

Reply:
it's design make it 100, I'm adjust to 100.r